src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -75,6 +75,48 @@
 
         return defaultBsPopperConfig;
     };
+    const getTextWidth = (text, fontProps = {}) => {
+        const styleProps = {
+            position: 'absolute',
+            left: '-1000px',
+            'white-space': 'nowrap',
+            ...fontProps,
+        };
+        const tag = doc.createElement('div');
+
+        tag.innerHTML = text;
+
+        for (const key in styleProps) {
+            tag.style[key] = styleProps[key];
+        }
```

Comment:
```suggestion
        tag.style = {
            ...tag.style,
            position: 'absolute',
            left: '-1000px',
            'white-space': 'nowrap',
            ...fontProps,
        }
```
maybe? or bu Object.assign